### DISCUSSION TITLE
About languages and translation

### DISCUSSION THREAD

**Discussion by Obijuan** [FPGAwars/icestudio#828] [Ideas]
I am writting here the suggestions made by @reclaimed in the pull request: #385  

*  move the language settings one level higher (View -> Language)
*  keep the English word "language" in the menu item name, like this: "Язык (language)"
*  the same with the language names: English, Русский (Russian), Ελληνικά (Greek), Deutsch (German)

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
